scripts/full-workflow.sh

```diff
@@ -0,0 +1,363 @@
+#!/usr/bin/env bash
+set -xeuf
+
+# This is the full workflow test of the ilab CLI.
+#
+# We expect it to be run anywhere `ilab` would run, including the instructlab
+# container images.
+#
+# It represents the workflow a typical user would run through with ilab.
+
+HF_TOKEN=${HF_TOKEN:-}
+GRANITE_SAFETENSOR_REPO="instructlab/granite-7b-lab"
+MIXTRAL_GPTQ_MODEL="TheBloke/Mixtral-8x7b-Instruct-v0.1-GPTQ"
+PROMETHEUS_7B_MODEL="prometheus-eval/prometheus-7b-v2.0"
```

Comment:
```suggestion
GRANITE_SAFETENSOR_REPO="instructlab/granite-7b-lab"
MIXTRAL_GPTQ_MODEL="TheBloke/Mixtral-8x7b-Instruct-v0.1-GPTQ"
PROMETHEUS_7B_MODEL="prometheus-eval/prometheus-7b-v2.0"
```
the naming is a little inconsistent here - maybe `GRANITE_SAFETENSOR_REPO` should be `GRANITE_7B_MODEL`?